src/controllers/user-data-manager.js

```diff
@@ -172,6 +206,13 @@ export default class UserDataManager {
     return [...remoteDataItems,...addToRemote]
   }
 
+  checkRequestQueue () {
+    if (this.requestsQueue.length > 0) {
```

Comment:
@irina060981 should this be in a while loop in case there is more than one request queued?

Reply:
No it shouldn't. The workflow here is the following:
At the satrt of each modifying method (update, deleteOne, deleteMany) it checks if some modifing method is currently in use (by this.blocking property).
If it is blocked, than it push a method execution parameters to the queue.
When the method finished to block (at the end of the execution), it executes checkRequestQueue. 
This method takes the first method from the queue and executes (and inside it the same blocking workflow is made).
This way checkRequestQueue will be executed each time the modifying method ends until queue becomes empty.

Reply:
ok, thanks for the clarification!